cmake/exodus.cmake

```diff
@@ -20,16 +22,18 @@ ExternalProject_Add(${EXODUS_NAME}
    -DCMAKE_Fortran_FLAGS:STRING=${CMAKE_Fortran_FLAGS}
    -DBUILD_SHARED:BOOL=${BUILD_SHARED_LIBS}
    -DBUILD_TESTING:BOOL=OFF
-   -DNETCDF_NCDUMP:PATH=${CMAKE_INSTALL_PREFIX}/bin/ncdump
-   -DNETCDF_SO_ROOT:PATH=${CMAKE_INSTALL_PREFIX}/lib/shared
-   -DNETCDF_INCLUDE_DIR:PATH=${CMAKE_INSTALL_PREFIX}/include
+   -DNETCDF_NCDUMP:PATH=<INSTALL_DIR>/bin/ncdump
```

Comment:
You can get the NetCDF prefix with `ExternalProject_get_property(${NETCDF_NAME} INSTALL_DIR)`